<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Detec">
    <property name="title">
     <string>检测(&amp;J)</string>
    </property>
    <addaction name="PhotoDet_action"/>
    <addaction name="Video_action"/>
   </widget>
   <widget class="QMenu" name="menu_Recog">
    <property name="title">
     <string>识别(&amp;R)</string>
    </property>
    <addaction name="Train_action"/>
    <addaction name="Recog_action"/>
   </widget>
   <addaction name="menu_Detec"/>
   <addaction name="menu_Recog"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="PhotoDet_action">
   <property name="text">
    <string>图像检测(&amp;T)</string>
   </property>
  </action>
  <action name="Video_action">
   <property name="text">
    <string>视频检测(&amp;S)</string>
   </property>
  </action>
  <action name="Train_action">
   <property name="text">
    <string>训练(&amp;X)</string>
   </property>
  </action>
  <action name="Recog_action">
   <property name="text">
    <string>图片识别(&amp;P)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
